base/shell/cmd/lang/zh-CN.rc

```diff
@@ -398,18 +398,15 @@ VER [/C][/R][/W]\n\n\
   /C  显示作者。\n\
   /R  显示再发布信息。\n\
   /W  显示保证信息。"
-    STRING_VERSION_HELP2 " comes with ABSOLUTELY NO WARRANTY; for details\n\
-type: `ver /w'. This is free software, and you are welcome to redistribute\n\
-it under certain conditions; type `ver /r' for details. Type `ver /c' for a\n\
-listing of credits."
-    STRING_VERSION_HELP3 "\n This program is distributed in the hope that it will be useful,\n\
- but WITHOUT ANY WARRANTY; without even the implied warranty of\n\
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n\
- GNU General Public License for more details."
-    STRING_VERSION_HELP4 "\n This program is free software; you can redistribute it and/or modify\n\
- it under the terms of the GNU General Public License as published by\n\
- the Free Software Foundation; either version 2 of the License, or\n\
- (at your option) any later version.\n"
+    STRING_VERSION_HELP2 " 不承诺 *任何品质保证* ；请输入 `ver /w' 查看详情。\n\
+本软件为自由软件，您可以在一定的条件下自由传播该软件；\n\
+请输入 `ver /r' 查看详情。输入 `ver /c' 可查看制作人员的列表。"
+    STRING_VERSION_HELP3 "\n 传播本程序时，我们希望它对其他人有用，但不承诺 *任何品质保证*；\n\
+ 甚至不包括任何隐含的适售性或对某个目的的适用性。\n\
+ 请参阅 GNU 通用公共许可证获悉详情。"
+    STRING_VERSION_HELP4 "\n 本程序是一个自由软件，您可以在自由软件基金会发布的 GNU 通用公共许可证\n\
+ 第二版（如果您自己愿意，也可以是任何一个新的版本）规定的条款下自由地\n\
+ 传播和/或修改本软件。\n"
```

Comment:
And I plan to replace my original translation of this license notice to the existing one. What do you think?

Reply:
> zh-HK doesn't translate this; should we?

(off-topic) as a translator of zh-HK, I haven't touch any cmd related strings because the Chinese font for command prompt seems not working. 😂

Reply:
As you decided let's just leave it translated https://github.com/reactos/reactos/pull/3933#discussion_r702383949